src/main/java/com/aws/greengrass/dependency/EZPlugins.java

```diff
@@ -9,9 +9,19 @@
 import com.aws.greengrass.logging.impl.LogManager;
 import com.aws.greengrass.util.Utils;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+// FIXME: android: replace that temporary stub solution
+//  see https://klika-tech.atlassian.net/browse/GGSA-62
+#if ANDROID
+import com.aws.greengrass.dependency.android.FastClasspathScanner;
+import com.aws.greengrass.dependency.android.ClassAnnotationMatchProcessor;
+import com.aws.greengrass.dependency.android.ImplementingClassMatchProcessor;
+#else
 import io.github.lukehutch.fastclasspathscanner.FastClasspathScanner;
 import io.github.lukehutch.fastclasspathscanner.matchprocessor.ClassAnnotationMatchProcessor;
 import io.github.lukehutch.fastclasspathscanner.matchprocessor.ImplementingClassMatchProcessor;
+#endif  /* ANDROID */
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, delete these tokens

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.